modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/FrontendTokenDefinitionRegistryImpl.java

```diff
@@ -191,35 +200,37 @@ protected String getServletContextName(Bundle bundle) {
 		return webContextPath;
 	}
 
-	protected String getThemeId(Bundle bundle) {
+	protected List<String> getThemeIds(Bundle bundle) {
 		URL url = bundle.getEntry("WEB-INF/liferay-look-and-feel.xml");
 
 		if (url == null) {
-			return null;
+			return Collections.emptyList();
 		}
 
 		try {
+			List<String> themeIds = new ArrayList<>();
+
 			String xml = URLUtil.toString(url);
 
 			xml = xml.replaceAll(StringPool.NEW_LINE, StringPool.SPACE);
 
 			Matcher matcher = _themeIdPattern.matcher(xml);
 
-			if (!matcher.matches()) {
-				return null;
-			}
+			while (matcher.find()) {
+				String themeId = matcher.group(1);
 
-			String themeId = matcher.group(1);
+				String servletContextName = getServletContextName(bundle);
```

Comment:
nit: If this is a constant value we can move it outside of the loop

Reply:
👍